core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -686,6 +686,8 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
               $scope.TAG_REGEX = GLOBALS.TAG_REGEX;
 
               $scope.CATEGORY_LIST_FOR_SELECT2 = [];
+              //Empty category added to force dropdown to scroll top
```

Comment:
Also, nit: comments should start with "// " (add a space after the slashes), and end with a full stop.